src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import './styles/reset.css';
+import Nav from './components/Nav';
+import Footer from './components/Footer';
+import Header from './components/Header';
+import CardList from './components/CardList';
+import useSampleData from './hooks/useSampleData';
 
 function App() {
+  const { links, name, owner } = useSampleData('folder') || {};
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Nav />
+      <Header folderName={name} owner={owner} />
+      <CardList cardsData={links} />
+      <Footer />
     </div>
```

Comment:
레이아웃 구성 상 의도적으로 작성하신 게 아니라면, 최상위 부모 태그에 아무런 스타일이 없을 땐 Fragment로 감싸주시면 됩니다.
Fragment는 실제 렌더링 시 DOM에 그려지지 않기 때문에 성능 상 (미약하게나마)유리합니다!
```suggestion
    <>
      <Nav />
      <Header folderName={name} owner={owner} />
      <CardList cardsData={links} />
      <Footer />
    </>
```